presto-main/src/test/java/com/facebook/presto/resourcemanager/TestResourceManagerClusterStatusSender.java

```diff
@@ -112,7 +112,7 @@ public void testNodeStatus()
                 format("Expect number of heartbeats to fall within target range (%s), +/- 50%%.  Was: %s", TARGET_HEARTBEATS, nodeHeartbeats));
```

Comment:
@tdcmeehan : Do you feel this check should be changed as well to avoid the comparisons using TARGET_HEARTBEATS to make the code more robust ?